pandas/core/strings.py

```diff
@@ -2114,19 +2114,55 @@ def rjust(self, width, fillchar=' '):
 
     def zfill(self, width):
         """
-        Filling left side of strings in the Series/Index with 0.
-        Equivalent to :meth:`str.zfill`.
+        Pad strings in the Series/Index by prepending '0' characters.
+
+        Strings in the Series/Index are padded with prepending '0' characeters
+        (i.e. on the left of the string) to reach a total string length
+        of `width`.  in the Series/Index with length greater than `width`
+        are unchanged.
+
+        Note: Differs from :meth:`str.zfill` which has special handling
+        for '+'/'-' in the string.
 
         Parameters
         ----------
         width : int
-            Minimum width of resulting string; additional characters will be
-            filled with 0
+            Minimum length of resulting string; strings with length less
+            than `width` be prepended with '0' characters.
 
         Returns
         -------
         filled : Series/Index of objects
```

Comment:
you can get rid of the `filled` and just leave the type (it was usual to name the return, but it was decided to not name them anymore as it doesn't add much value).